library/core/src/ptr/const_ptr.rs

```diff
@@ -767,8 +764,9 @@ impl<T: ?Sized> *const T {
     /// // This would be incorrect, as the pointers are not correctly ordered:
     /// // ptr1.sub_ptr(ptr2)
     /// ```
-    #[unstable(feature = "ptr_sub_ptr", issue = "95892")]
-    #[rustc_const_unstable(feature = "const_ptr_sub_ptr", issue = "95892")]
+    #[stable(feature = "ptr_sub_ptr", since = "CURRENT_RUSTC_VERSION")]
+    #[rustc_const_stable(feature = "const_ptr_sub_ptr", since = "CURRENT_RUSTC_VERSION")]
+    #[rustc_allow_const_fn_unstable(core_intrinsics)]
```

Comment:
We never "allow" `core_intrinsics`. Instead, the corresponding intrinsic should get marked as const-stable. This requires a t-lang FCP.

Reply:
@bend-n I am curious, why did you use? `rustc_allow_const_fn_unstable`. That is a "nuclear option" attribute that should hardly ever be used... so if you saw this in some docs or so, maybe we have to word them more strongly.

Reply:
@nora told me to
> allow internal const unstable however its called

how should i do it?

Reply:
> i can const stab the actual intrinsic?

Yes. Please read the docs at the top of `intrinsics/mod.rs`. You can use `ptr_offset_from` as an example.

Reply:
so just add the const_indirect thing, yeah?
i think i did that right then?

Reply:
Yes, looks good :)

Reply:
You just pinged a random user called "nora" :D. But I see, makes sense.